test/detect/cpp/comment.txt

```diff
@@ -1,16 +0,0 @@
-/*
-To use this program, compile it -- if you can -- and then type something like:
-
-chan -n 5000 -d 2 < input.txt
-
-In this case, it will produce 5000 words of output, checking two-word groups.
-(The explanation above describes two-word generation. If you type "-d 3",
-the program will find three-word groups, and so on. Greater depths make more
-sense, but they require more input text and take more time to process.)
-
-http://www.eblong.com/zarf/markov/
-*/
-
-
-/* make cpp win deterministically over others with C block comments */
-cout << endl;
```

Comment:
This is a terrible detect test being nothing but comments.. we shouldn't expect to necessarily get this right anyways, so removing.

Reply:
Yes I agree.